### ISSUE TITLE
Make missing in v1.32.1 Docker image

### ISSUE THREAD

**Issue by invidian** [golangci/golangci-lint#1483]
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

It seems Docker images up to `golangci/golangci-lint:v1.32.0` were containing `make` binary, which was handy, when having a `Makefile` target which will trigger linting, which I believe might be a common scenario.

However, `golangci/golangci-lint:v1.32.1` image no longer includes `make`, which at the moment breaks the CI for our project and blocks us from updating to latest version of the linter.

It would be great to have `make` back, so we can keep using official upstream images for linting!

Console log:
```console
$ docker run -it --entrypoint /bin/sh golangci/golangci-lint:v1.32.0 -c 'which make'
Unable to find image 'golangci/golangci-lint:v1.32.0' locally
v1.32.0: Pulling from golangci/golangci-lint
e4c3d3e4f7b0: Already exists
101c41d0463b: Already exists
8275efcd805f: Already exists
751620502a7a: Already exists
aaabf962c4fc: Already exists
7883babec904: Pull complete
1791d366c848: Pull complete
974f06baf804: Pull complete
Digest: sha256:d69fe1a8460770ddca730654c880a9ddad33bc538f162be084135d73d7adf8bd
Status: Downloaded newer image for golangci/golangci-lint:v1.32.0
/usr/bin/make
$ docker run -it --entrypoint /bin/sh golangci/golangci-lint:v1.32.1 -c 'which make'
Unable to find image 'golangci/golangci-lint:v1.32.1' locally
v1.32.1: Pulling from golangci/golangci-lint
188c0c94c7c5: Pull complete
0ef7d3d256c8: Pull complete
de9db76c5a1d: Pull complete
0eba1c9be4d2: Pull complete
0d57e429df01: Pull complete
59a968b239ef: Pull complete
df2d921a9792: Pull complete
Digest: sha256:5a0752732d0da7b838db12c317fffb76b0a13ef594ced9ceea32dc4def48f5b6
Status: Downloaded newer image for golangci/golangci-lint:v1.32.1
```

**Comment by sayboras** [comment]
Thanks for reporting this issue, recent PR was done by me https://github.com/golangci/golangci-lint/pull/1471, but I can't see any change related to make. My guess right now is that upstream docker image didn't ship with make anymore. 

Just curious if you are planning to work on this :smile: ?

**Comment by invidian** [comment]
Yeah, I looked at the diff between v1.32.0 and v1.32.1 and it doesn't seem to be a regression here. I still decided to report here to let you guys know as other people may be affected by this too. I'l digging deeper right now :pick:

**Comment by invidian** [comment]
Oh, interesting: 
```console
$ docker run -it --entrypoint /bin/sh golangci/golangci-lint:v1.32.1 -c 'cat /etc/os-release'
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.12.1
PRETTY_NAME="Alpine Linux v3.12"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
$ docker run -it --entrypoint /bin/sh golangci/golangci-lint:v1.32.0 -c 'cat /etc/os-release'
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

I suppose this was not intended.

**Comment by sayboras** [comment]
good finding :100:, now I can confirm it's my mistake overriding the tag for docker image. I used the same tags for both :facepalm:.

Thanks for checking on this :100: 

```
          tags: |	
            golangci/golangci-lint:${{ steps.prepare.outputs.tag_name }}-alpine	
            golangci/golangci-lint:${{ steps.prepare.outputs.major_tag }}-alpine	
            golangci/golangci-lint:latest-alpine
```

**Comment by invidian** [comment]
No worries, it didn't take too much time. @sayboras are you going to fix it then?

**Comment by sayboras** [comment]
> No worries, it didn't take too much time. @sayboras are you going to fix it then?

Yeah, I can do it, but later in the day only, I am about to start my day job :)

**Comment by aaronchipper** [comment]
Thanks for this.
It bit us on the backside over the weekend when pushing a deploy. We worked around it adding a make install to the build scripts, but it definitely was a surprise!
Let me know if you want someone to test and confirm - happy to do so.

**Comment by sayboras** [comment]
Thanks guys for offering help to test this one out, the changes are available in #1487, please let me know if there is anything that I could have missed (again :sweat: )

**Comment by aaronchipper** [comment]
Looking good. Thanks for this.
